Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "CompatHelper"
 uuid = "aa819f21-2bde-4658-8897-bab36330d9b7"
 authors = ["Dilum Aluthge", "Brown Center for Biomedical Informatics", "contributors"]
-version = "3.11.0"
+version = "4.0.0"
```

Comment:
I think this is a bugfix, right? I don't think we need to bump the major version. This was always supposed to be how CompatHelper works.

Reply:
Hmm I guess it depends on whether you consider the individual function signatures to form part of the public API (=> major), or if it's just the overall effect of running the main function (=> bugfix). Happy to go either way on this 🙂

Reply:
I also think we can get away without modifying any function signatures, at least not of public functions.

Reply:
Gotcha!